src/components/biggive-campaign-card-filter-grid/biggive-campaign-card-filter-grid.tsx

```diff
@@ -143,6 +143,7 @@ export class BiggiveCampaignCardFilterGrid {
       filterLocation: this.selectedFilterLocation,
     };
 
+    // @ts-ignore
```

Comment:
I don't like using @ts-ignore, but on old code that we may not really need to edit or even read any time soon it might be preferable to spending time finding a nicer solution.